README.md

```diff
@@ -0,0 +1,235 @@
+or added latency to create uncertainty for network observers. This
+uncertainty is in regards to trying to link incoming messages with the
+outgoing messages.
+
+
+# Project Status
+
+Many excited new changes are forthcoming!
+You can watch our progress being tracked, here:
+
+https://github.com/orgs/katzenpost/projects/6/views/5
+
+
+There are essentially two types of interaction with a Katzenpost mixnet:
+1. clients talk to mixnet services and their traffic stays in the mixnet
+2. clients talk to Internet services; proxying through the mixnet onto the Internet.
+
+# Client:
+
+Currently, [Katzen](https://github.com/katzenpost/katzen) is the only client available for use with Katzenpost. However a SOCKS proxy client is forthcoming
+and you'll be able to use that with many existing applications.
+
+# Server Side: Easy One Step BUILD
+
+You'd want to build the server side if you are
+a developer, mix node operator or dirauth operator.
+
+This builds 3 binaries:
+
+* mix server
+* dirauth server
+* mixnet ping CLI command
+
+All you have to do is type:
+
+```bash
+make
+```
+
+For Docker users, you can use:
+
+```bash
+make docker
+```
+
+# Server Side Usage/Configuration
+
+Our docker configuration is the most comprehensive and up to date
+place to learn about how to configure a Katzenpost mix network. Run
+the makefile in the docker directory to get a usage menu:
+
+```bash
+$ cd katzenpost/docker; make 
+These make targets allow you to control the test network:
+ run                - run the testnet in the foreground, until ctrl-C
+ start              - start the testnet in the background
+ stop               - stop the testnet
+ wait               - wait for testnet to have consensus
+ watch              - tail -F all logs
+ status             - show testnet consensus status
+ show-latest-vote   - does what it says
+ run-ping           - send a ping over the testnet
+ clean-local        - stop, and delete data and binaries
+ clean-local-dryrun - show what clean-local would delete
+ clean              - the above, plus cleans includes go_deps images
+```
+
+**You can run a docker mixnet locally and then inspect the configuration files
+to learn how to configure a Katzenpost mixnet.**
+
+
+Documentation is a work in progress:
+
+* [mix server docs](docs/handbook/mix_server.rst)
+
+* [dirauth server docs](docs/handbook/voting_pki.rst)
+
+
+
+# Expert's Corner
+
+Katzenpost is an unverified decryption mix network that uses a continuous time
+mixing strategy with client selected exponential delays and a stratified (layered) topology. 
+
+Our documentation is in progress, but we have some resources for experts:
+
+* Our threat model document, work-in-progress, can be found [here.](https://raw.githubusercontent.com/katzenpost/katzenpost/add_threat_model_doc/docs/specs/threat_model.rst)
+
+* Our design specification documents are available [here.](https://github.com/katzenpost/katzenpost/tree/main/docs/specs)
+
```

Comment:
somewhere in here it would be good to mention the existing client libraries and any other CLI tools that we've got (I guess they're not yet merged, though), but for example the memspool client/server or panda client/server are all re-usable components that developers could use today. We might put this section in the "Experts' area below, though.